src/webroot/js/cake-core.js

```diff
@@ -0,0 +1,25 @@
+//$(document).ready(function () {
+  var elements = document.getElementsByClassName('cake-core-postLink');
+
+  var postLinkFunction = function (event) {
+    var formName = this.getAttribute("data-cake-core-form");
+    var confirmMessage = this.getAttribute('data-cake-core-confirm');
+    if (confirmMessage !== null && confirmMessage !== 'undefined') {
+      if (confirm(confirmMessage)) {
+        document[formName].submit();
+        return false;
+      }
+      else {
```

Comment:
Oh yeah, 4 spaces. The file was in my own application as I had to test it and I am using just 2 spaces. Therefore the different code style. :wink: